cmake/archive_options.cmake

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(APPLE)
+  # When building on Apple platforms, ranlib complains about file has no
```

Comment:
"about file has no" => "about files that have no" ?

Reply:
I should have put this in quotes because it's the actual error message from ranlib. I've adjusted this and some of the other comments to make this clearer.